test/functional/fixtures/api/es-next/press-key/testcafe-fixtures/press-key-test.js

```diff
@@ -0,0 +1,20 @@
+// NOTE: to preserve callsites, add new tests AFTER the existing ones
+import { Hybrid } from 'testcafe';
+import { expect } from 'chai';
+
+
+fixture `PressKey`
+    .page `http://localhost:3000/api/es-next/press-key/pages/index.html`;
+
+const focusInput    = Hybrid(() => document.getElementById('input').focus());
+const getInputValue = Hybrid(() => document.getElementById('input').value);
+
+test('Press keys', async t => {
+    await focusInput();
```

Comment:
I'm not sure that selector for the `pressKey` action is a nice idea. What does `pressKey(selector, keys)` mean, `click(selector) + press(keys)`? I think it's more clear to use `t.click().press()` in this case. I believe `pressKey` with just `keys` argument is more useful for some cases (test control keyboard navigation for example).